templates/index.phtml

```diff
@@ -13,6 +13,30 @@
     <title>Home | Your place</title>
 </head>
 <body>
+<div class="main">
+    <div class="logo">
+        <a href="#">
+            <img src="./img/logo_4.png" alt="Awesome Badger logo">
+        </a>
+        <h1>Awesome badger bookings</h1>
+    </div>
+    <div class="date_range_picker">
+        <form action="#">
+            <div class="form_content">
+                <input type="text" id="from" name="from" placeholder="From...">
+                <input type="text" id="to" name="to" placeholder="To...">
+                <input type="submit" class="date_submit">
```

Comment:
need to change the date so there is a minimum stay.

Reply:
please use the word 'search'

Reply:
fixed in 3resultpage